common/Makefile

```diff
@@ -1,2 +1,6 @@
 test:
-	go test -v ./...
\ No newline at end of file
+	go test -v ./...
+
+genrest:
```

Comment:
Does this genrest belongs to common package?

Reply:
No, its leftover, will move it.

Reply:
Done